src/components/Todo/EditCard/edit-action-bar.component.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import PropTypes from "prop-types";
+import deleteButton from "assets/images/delete.png";
+import checkButton from "assets/images/check.png";
+import { COMPLETE_BUTTON, DELETE_BUTTON } from "common/constants";
+
+const EditActionBar = ({
+  todo,
+  editedTask,
+  handleEditTask,
+  handleCompleteTask,
+  handleEditCancelTask,
```

Comment:
callback props should have a `on` prefix.

```suggestion
  onEditTask,
  onCompleteTask,
  onCancelEditTask,
```